files/settings-default.cfg

```diff
@@ -216,6 +216,9 @@ followers attack on sight = false
 # Can loot non-fighting actors during death animation
 can loot during death animation = true
 
+# Makes the value of filled soul gems dependent only on soul magnitude (with formula from the Morrowind Code Patch)
+rebalance soul gem values = false
```

Comment:
Honestly, perhaps we should make this true since this is something that should have been fixed in Morrowind but wasn't.